web/client/components/geostory/contents/Contents.jsx

```diff
@@ -72,6 +73,7 @@ export default ({
                     {...contentProps}
                     {...props}
                     storyTheme={storyTheme}
+                    defaultThemeForType={overrideThemeTypeWithDefault}
```

Comment:
Please remove `defaultThemeForType` and `overrideThemeTypeWithDefault` from here and from other components in this PR, they are not needed anymore.